tests/test_fields.py

```diff
@@ -875,6 +875,106 @@ def test_list_of_raw(self):
         self.assert_field(field, data, data)
 
 
+class WildcardFieldTest(BaseFieldTestMixin, FieldTestCase):
+    field_class = partial(fields.Wildcard, fields.String)
+
+    def test_defaults(self):
+        field = fields.Wildcard(fields.String)
+        assert not field.required
+        assert field.__schema__ == {'type': 'array', 'items': {'type': 'string'}}
```

Comment:
Hum, I don't think the resulting data is an array. I think we need to make use of `additionalProperties`. See my review comment for details.